engine/utils/string_utils.h

```diff
@@ -13,6 +13,11 @@ struct ParsePromptResult {
   std::string ai_prompt;
 };
 
+inline bool EqualsIgnoreCase(const std::string& a, const std::string& b) {
```

Comment:
It would be great if we have unit tests for this function

Reply:
yeah, let me add it into another PR, which comes right after this